components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -163,7 +163,7 @@ func main() {
 		"BackupInt":               runtime.NewGenericComponentDisabler("backup-init", "kyma-system"),
 		"Backup":                  runtime.NewGenericComponentDisabler("backup", "kyma-system"),
 		"KnativeProvisionerNatss": runtime.NewGenericComponentDisabler("knative-provisioner-natss", "knative-eventing"),
-		"NatssStreaming":          runtime.NewGenericComponentDisabler("nats-streaming", "natss"),
```

Comment:
Here we need to add kafka charts, to make them optional.

Reply:
Done